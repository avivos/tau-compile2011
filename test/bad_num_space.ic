int legal1 = -1214000003
int legal2 = 2049999999
int legal3 = -    2147483648
int illegal = 2147489999
