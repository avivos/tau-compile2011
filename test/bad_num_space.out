1: INT
1: ID(legal1)
1: ASSIGN
1: MINUS
1: INTEGER(1214000003)
2: INT
2: ID(legal2)
2: ASSIGN
2: INTEGER(2049999999)
3: INT
3: ID(legal3)
3: ASSIGN
3: MINUS
3: INTEGER(2147483648)
4: INT
4: ID(illegal)
4: ASSIGN
Line 4: Integer out of range: 2147489999
